src/app/f2f/controllers/addressResults.js

```diff
@@ -37,9 +31,11 @@ class AddressResultsController extends BaseController {
     super.saveValues(req, res, () => {
       try {
         const selectedAddress = req.form.values.addressResults;
+        console.log("selectedAddress", selectedAddress);
```

Comment:
This should be logger rather than console.
Also can we make these debug as that would be PII